src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -19,17 +19,69 @@ const StatCard = ({ number, secondaryNumber, statement }: StatCardProps) => (
   </div>
 );
 
+type AxisProp = {
+  x: string | number;
+  y: number;
+};
+
+type BarStatProps = {
+  axes: AxisProp[];
+  legend: { title: string; subtitle: string };
```

Comment:
I think this is not quite a "legend", since it doesn't really map symbols, colors, or other graphical hints to their meanings in the graph. I can't really think of a good name for the combination of both this title and subtitle, so it might be better to just remove the "legend" part of it and just make "title" and "subtitle" be direct properties of `BarStatProps`.